core/swiftAction/Dockerfile.hold2

```diff
@@ -1,27 +0,0 @@
-FROM ubuntu:14.04
```

Comment:
If this file gets created then deleted, you should probably squash some commits to make these changes disappear from the history.